base_agent/condition.py

```diff
@@ -52,6 +73,33 @@ def check(self):
         return False
 
 
+class TaskStatusCondition(Condition):
+    def __init__(self, agent, task_memid, status="finished"):
+        super().__init__(agent)
+        self.status = status
+        self.task_memid = task_memid
+
+    def check(self):
```

Comment:
This returns true when the task is finished, running or paused and never returns False (since it will raise an assertion error before that)? Why is that?

Reply:
lol its a bug